grails-app/services/org/broadinstitute/orsp/QueryService.groovy

```diff
@@ -911,14 +911,19 @@ class QueryService implements Status {
                 }
 
                 // Exclude abandoned by default unless statuses are specifically requested
-                if (options.ignoreAbandoned && !options.issueStatusNames) {
+                if (options.ignoreAbandoned && !options.issueStatusNames && !options.freeText) {
                     ne("status", IssueStatus.Abandoned.name)
                 }
 
                 if (options.projectKey) {
                     // For backwards compatibility with existing "ORSP" prefixes, ignore the prefix and like-clause on the identifier
                     like("projectKey", getIssueNumberFromString(options.projectKey))
                 }
+
```

Comment:
Let's review the behavior of withCriteria in this case. This method have 2 steps : 1) perform a query with some options, 2) filter results with some options too. Not sure if step 1 will succeed for term != project keys.